protected/humhub/commands/RectorController.php

```diff
@@ -64,7 +64,7 @@ private function process(string $args = ''): int
         }
 
         $commands = [
-            'cd ' . Yii::getAlias('@webroot'),
+            'cd ' . Yii::getAlias('@config'),
```

Comment:
@yurabakhtin I like the fact that `rector.php` is no longer in the root directory.

Maybe we can even move it to `/protected/humhub/config`? Because `/protected/config/` is more for custom configuration.

Reply:
@luke- Done in commit https://github.com/humhub/humhub/pull/6048/commits/c6a9756be644d3d0fddd505960feff3d22d8e136.